```
(ssh)root@Hydaelyn [ s ] # sqlmap -r 1.txt 
        ___
       __H__
 ___ ___[.]_____ ___ ___  {1.9.5#stable}
|_ -| . ["]     | .'| . |
|___|_  ["]_|_|_|__,|  _|
      |_|V...       |_|   https://sqlmap.org

[!] legal disclaimer: Usage of sqlmap for attacking targets without prior mutual consent is illegal. It is the end user's responsibility to obey all applicable local, state and federal laws. Developers assume no liability and are not responsible for any misuse or damage caused by this program

[*] starting @ 20:54:03 /2025-06-04/

[20:54:03] [INFO] parsing HTTP request from '1.txt'
[20:54:03] [INFO] testing connection to the target URL
got a 301 redirect to 'https://srcblog.neko25.top/usr/themes/G-master/static/css/G.css?v=3.4.1'. Do you want to follow? [Y/n] 
[20:54:04] [ERROR] user quit
```

```
(ssh)root@Hydaelyn [ s ] ! sqlmap -u "https://srcblog.neko25.top/?id=1"
        ___
       __H__
 ___ ___[']_____ ___ ___  {1.9.5#stable}
|_ -| . [.]     | .'| . |
|___|_  [,]_|_|_|__,|  _|
      |_|V...       |_|   https://sqlmap.org

[!] legal disclaimer: Usage of sqlmap for attacking targets without prior mutual consent is illegal. It is the end user's responsibility to obey all applicable local, state and federal laws. Developers assume no liability and are not responsible for any misuse or damage caused by this program

[*] starting @ 20:53:45 /2025-06-04/

[20:53:46] [INFO] testing connection to the target URL
[20:53:46] [WARNING] potential permission problems detected ('Access denied')
[20:53:46] [WARNING] the web server responded with an HTTP error code (403) which could interfere with the results of the tests
[20:53:46] [INFO] checking if the target is protected by some kind of WAF/IPS
[20:53:46] [WARNING] HTTP error codes detected during run:
403 (Forbidden) - 1 times


[*] ending @ 20:53:46 /2025-06-04/
```

working well

--- a/app-penetration/sqlmap/autobuild/overrides/usr/bin/sqlmap
+++ b/app-penetration/sqlmap/autobuild/overrides/usr/bin/sqlmap
@@ -1,3 +1,3 @@
 #!/bin/sh
-cd /usr/share/sqlmap
-python sqlmap.py "$@"
+
+python3 /usr/share/sqlmap/sqlmap.py "$@"

--- a/app-penetration/sqlmap/autobuild/overrides/usr/bin/sqlmapapi
+++ b/app-penetration/sqlmap/autobuild/overrides/usr/bin/sqlmapapi
@@ -1,3 +1,3 @@
 #!/bin/sh
-cd /usr/share/sqlmap
-python sqlmapapi.py "$@"
+
+python3 /usr/share/sqlmap/sqlmapapi.py "$@"

--- a/app-penetration/sqlmap/spec
+++ b/app-penetration/sqlmap/spec
@@ -1,4 +1,4 @@
-VER=1.8.8
+VER=1.9.5
 SRCS="git::commit=tags/$VER::https://github.com/sqlmapproject/sqlmap"
 CHKSUMS="SKIP"
 CHKUPDATE="anitya::id=14901"

